packages/adapter-vercel/index.js

```diff
@@ -39,6 +39,32 @@ const plugin = function (defaults = {}) {
 			builder.rimraf(dir);
 			builder.rimraf(tmp);
 
+			if (fs.existsSync('./vercel.json')) {
+				const vercel_file = fs.readFileSync('./vercel.json', 'utf-8');
+				const vercel_config = JSON.parse(vercel_file);
+				const crons = /** @type {Array<unknown>} */ (
+					Array.isArray(vercel_config?.crons) ? vercel_config.crons : []
+				);
+				const GET_routes = builder.routes.filter((route) => route.methods.includes('GET'));
```

Comment:
I wonder if we should add `route.api` and `route.page` properties (or something) so that we can screen out pages, since they have `GET` but don't qualify as cron tasks

Reply:
actually maybe `route.server` would be better than `route.api`, so that it matches `+server` and `+page`. these properties would both be booleans

Reply:
jokes on us, I think we actually had an api like this at some point but removed it because there was no use case